charts/sourcegraph/templates/otel-collector/otel-agent.DaemonSet.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: apps/v1
```

Comment:
I think we should inherit the previous `tracing.enabled` flag to allow users to turn off otel collector if they wish

Reply:
What is the default for `tracing.enabled`? If we provide a toggle I might prefer this option to be deprecated in favour of e.g.

```
opentelemetry.enabled (default true)
jaeger.enabled (default false)
```

Reply:
yeah, it makes more sense to use `opentelemetry`!

but le'ts make sure we document this in the changelog since it's a pretty big change

Reply:
Thanks for your feedback @michaellzc 🥇 

The changelog is updated here https://github.com/sourcegraph/sourcegraph/pull/41241#pullrequestreview-1094537210